Source/Scene/Instanced3DModel3DTileContent.js

```diff
@@ -419,8 +419,9 @@ define([
             instanceScale.x = 1.0;
             instanceScale.y = 1.0;
             instanceScale.z = 1.0;
-            var scale = featureTable.getProperty('SCALE', i, ComponentDatatype.FLOAT);
+            var scale = featureTable.getProperty('SCALE', i, ComponentDatatype.FLOAT, 1);
             if (defined(scale)) {
+                if (Array.isArray(scale)) scale = scale[0];
```

Comment:
If we want to enforce a stricter interpretation, I would say just keep SCALE as a per-instance semantic. It would make the standard more difficult to interpret if we add two different types. I'd rather keep it simple.

I pointed it out because currently, setting a scalar value to SCALE would apply the scaling to all instances.